src/components/Navigation.js

```diff
@@ -1,18 +1,19 @@
-import React from 'react';
+import React from "react";
+import { Link } from "react-router-dom";
 
 const Navigation = () => {
   return (
     <div>
       <div className="App">
         <h1>React Router Mini</h1>
         <div>
-          <a href="">Home</a>
+          <Link to="/">Home</Link>
```

Comment:
Pop Quiz: What's the difference between Link and NavLink?

Reply:
NavLink includes props to style the selected route